src/kits/tracker/FindPanel.cpp

```diff
@@ -253,7 +256,20 @@ FindWindow::FindWindow(const entry_ref* newRef, bool editIfTemplateOnly)
 
 	fBackground = new FindPanel(fFile, this, fFromTemplate,
 		fEditTemplateOnly);
-	SetLayout(new BGroupLayout(B_VERTICAL));
+		
+	fMenuBar = new BMenuBar("Menu Bar");
```

Comment:
Why doesn't the BuildMenuBar() function not create the main menu bar?

Reply:
Fixed this in the latest commit.